tests/test_user_exercise.py

```diff
@@ -0,0 +1,38 @@
+from app.routers.user_exercise import create_user_exercise,\
+    does_user_exercise_exist, get_user_exercise
+from app.routers.user import create_user
+
+
+class TestUserExercise:
+
+    def test_create_user_exercise(self, session):
```

Comment:
Can you use the `user` fixation instead?

Reply:
No , Its make alot of issues,
I prefer saty as is